packages/kbn-crypto/BUILD.bazel

```diff
@@ -37,6 +37,7 @@ TYPES_DEPS = [
   "@npm//@types/node",
   "@npm//@types/node-forge",
   "@npm//@types/testing-library__jest-dom",
+  "@npm//@emotion/react",
```

Comment:
@mistic does this make sense to you?  Or do we think it's one of dependency ordering issues?

Reply:
There was some related discussion in https://github.com/elastic/kibana/pull/99382#discussion_r640078833
Basically, modifying `tsconfig.base.json` makes this a dependency almost everywhere

Reply:
yeah it makes sense as we are inheriting from the `tsconfig.base.json` here and we don't override the type definitions in tsconfig file of that package. probably we should do it